python_modules/libraries/dagster-dbt/packages/dagster/macros/dagster__log_columns_in_relation.sql

```diff
@@ -0,0 +1,11 @@
+{% macro dagster__log_columns_in_relation() %}
```

Comment:
are double underscores a convention used elsewhere?

Reply:
https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch

This convention is usually used for adapters. But our package isn't an adapter package. I can remove it.

In https://github.com/dagster-io/dagster/pull/19628/files#diff-40533874f1a224209259f37e172425f8b620b733187e9aa92e4d203f6c0d7292R555-R563, it's already invoked in the `dagster` namespace anyways, so this is overkill.

Reply:
Makes sense

Reply:
Updated